tests/page/page-click.spec.ts

```diff
@@ -38,22 +38,6 @@ it('should click button inside frameset', async ({ page, server }) => {
   expect(await frame.evaluate('result')).toBe('Clicked');
 });
 
-it('should click a button that closes popup', async ({ page }) => {
-  it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/20093' });
```

Comment:
Should we just skip it in Firefox instead?

Reply:
I don't think we want to commit to this behavior in any of the browsers.